extensions/wikia/PortableInfobox/styles/PortableInfoboxMixins.scss

```diff
@@ -71,7 +69,7 @@
 		&:not(:first-child) {
 			@include flex-basis($infobox-width * 2 / 3);
 
-			padding-left: $padding;
+			padding-left: 10px;
```

Comment:
Try to avoid magic numbers. 
IMO having it in variable was better.

Except that LGTM